molecule/default/converge.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Converge
+  hosts: all
+  become: true
+
+  vars:
+    solr_service_name: solr
+    solr_home: /var/solr
+    solr_core_name: test
+    solr_core_conf_source: "{{ solr_home }}/data/{{ solr_core_name }}/conf"
+    solr_core_conf_dest: "{{ solr_home }}/data/{{ solr_core_name }}/conf"
+  pre_tasks:
+
+    - name: Create solr conf directory
+      file:
+        path: /var/solr/data/test/conf
+        state: directory
+
+    - name: Install rsync
```

Comment:
This is redundant. Rsync is already being installed locally (on the controller) in the 'prepare' playbook.

Reply:
@Chithrak07 remove this task. I dont know why it fails, maybe its trying to run dnf instead of yum, but either way it's not needed here.

Reply:
@dale-c-anderson : Did you mean to remove the creation of solr core directory? Isn't that a requirement?